geps/gep-1494/index.md

```diff
@@ -0,0 +1,131 @@
+# GEP-1494: HTTP Auth in Gateway API
+
+* Issue: [#1494](https://github.com/kubernetes-sigs/gateway-api/issues/1494)
+* Status: Provisional
+
+(See status definitions [here](/geps/overview/#gep-states).)
+
+
+## TLDR
+
+Provide a method for configuring **Gateway API implementations** to add HTTP Authentication for north-south traffic. The method may also include Authorization config if practical.
```

Comment:
Hmm.. Are these auth* applicable only to inbound traffic, or do they cover both ingress and egress?

Reply:
Theoretically, it could cover both, depending on how you orient the Gateway, but I've added something to say this is only ingress for now. Good point, thanks!